### PR TITLE
Added safe area documentation for UI

### PR DESCRIPTION
**Pull request by yaustar**
Related ticket: https://github.com/playcanvas/editor/issues/506

Linked project: https://playcanvas.com/project/828118/overview/mobile-ui-safe-areas

image

### CODE REVIEW

File: content/en/user-manual/user-interface/safe-area.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+However, when opened on a mobile device such as the iPhone X, the 'Left' text is rendered under the notch and the 'Bottom' text is rendered under the navigation bar.
+
+<img src="/images/user-manual/user-interface/safe-area/mobile-view-render-under-notch.png" width="500px">
+
+## Safe Area
+
+To help developers, browsers on the these devices do support [environment variables][env-mdn] in CSS to return values for positioning elements within an area that is occupied by the notch or navigation bar. This is known as a 'safe area'.
+
+We have a [project with a reusable script][safe-area-project] that takes those CSS values and applies them to an UI Element Group entity via resizing the margins.
```
Comment (yaustar) [inline content/en/user-manual/user-interface/safe-area.md:25]:
```suggestion
We have a [project with a reusable script][safe-area-project] that takes those CSS values and applies them to an UI Group Element entity via resizing the margins.
```

File: content/en/user-manual/user-interface/safe-area.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+<img src="/images/user-manual/user-interface/safe-area/mobile-view-safe-area.png" width="500px">
+
+The UI setup in the project has an Entity with a full screen Element Group named 'Safe Area'. This has the script 'mobileSafeArea' attached which contains the logic for fitting the Element within the safe area of the device.
```
Comment (yaustar) [inline content/en/user-manual/user-interface/safe-area.md:29]:
```suggestion
The UI setup in the project has an Entity with a full screen Group Element named 'Safe Area'. This has the script 'mobileSafeArea' attached which contains the logic for fitting the Element within the safe area of the device.
```

**Comment by mvaligursky** [comment]
Looks great. Do we have a license to use iphone-notch.jpg ?

**Comment by yaustar** [comment]
> Looks great. Do we have a license to use iphone-notch.jpg ?

Good point 🤔 Let me see if I can find one that's CC0. That existing one is from Apple.

**Comment by yaustar** [comment]
@mvaligursky Added a CC 4.0 image from Wikipedia

**Review by mvaligursky — APPROVED** [review]

**Pull request merged**